.travis.yml

```diff
@@ -16,3 +16,5 @@ git:
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("StatsBase"); Pkg.test("StatsBase"; coverage=true)'
 after_success:
   - julia -e 'cd(Pkg.dir("StatsBase")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'Pkg.add("Documenter")'
+  - julia -e 'cd(Pkg.dir("StatsBase")); include(joinpath("docs", "make.jl"))'
```

Comment:
Might as well make this one line in keeping with the coverage line item and add the package in the same step as building the docs. Doesn't really matter though.